src/components/presentational/SparkBarChart/SparkBarChart.tsx

```diff
@@ -8,24 +8,46 @@ export interface SparkBarChartProps {
 	bars: SparkBarChartBar[];
 	innerRef?: React.Ref<HTMLDivElement>;
 	style?: React.CSSProperties
+	gap?: number;
+	variant?: "square" | "rounded";
 }
 
 export interface SparkBarChartBar {
 	color: ColorDefinition;
 	barFillPercent: number;
+	opacity?: number;
 }
 
 export default function SparkBarChart(props: SparkBarChartProps) {
```

Comment:
Minor: save `prop` reference below.
```suggestion
export default function SparkBarChart({averageFillPercent, bars, innerRef, style, gap, variant}: SparkBarChartProps) {
```